x/crosschain/keeper/evm_hooks_test.go

```diff
@@ -0,0 +1,96 @@
+package keeper_test
+
+import (
+	"encoding/json"
+	"math/big"
+	"testing"
+
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/stretchr/testify/require"
+	"github.com/zeta-chain/zetacore/common"
+	crosschainkeeper "github.com/zeta-chain/zetacore/x/crosschain/keeper"
+)
+
+//
+
+func TestValidateZrc20WithdrawEvent(t *testing.T) {
+	t.Run("valid event", func(t *testing.T) {
+		btcMainNetWithdrawalEvent, err := crosschainkeeper.ParseZRC20WithdrawalEvent(*GetValidSampleBlock(t).Logs[3])
+		require.NoError(t, err)
+		err = crosschainkeeper.ValidateZrc20WithdrawEvent(btcMainNetWithdrawalEvent, common.BtcMainnetChain().ChainId)
+		require.NoError(t, err)
+	})
+	t.Run("invalid value", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("invalid amount", func(t *testing.T) {
```

Reply:
Id rather keep this value, as that's the name of the field in the event